tools/cldr/cldr-to-icu/README.txt

```diff
@@ -6,14 +6,34 @@
 Basic instructions for running the LdmlConverter via Maven
 ==========================================================
 
+Requirements
+------------
+
+* A CLDR release for supplying CLDR data and the CLDR API.
+* The Maven build tool
+* The Ant build tool
+
 Important directories
 ---------------------
 
-<CLDR_DIR>  = The root directory of the CLDR release.
+<CLDR_DIR>  = The root directory of the CLDR project.
```

Comment:
You may want to mention here or elsewhere that the CLDR directory in question should contain CLDR data in production form, not raw form.

Reply:
i.e., it should be cldr-staging.

Reply:
Ahh yes, this was in flux as I was originally doing this and it slipped through the cracks.